cmd/gen-go-sample/main.go

```diff
@@ -353,14 +353,38 @@ func (g *generator) genSample(sampConf schema_v1p2.Sample, methConf GAPICMethod)
 
 	p := g.pt.Printf
 
-	// Region tag, function signature and new client
+	// Region tag
 	argStr, err := argListStr(initInfo, g)
 	if err != nil {
 		return err
 	}
 	p("")
 	p("// [START %s]", sampConf.RegionTag)
 	p("")
+
+  // generate comments above the sample function
```

Comment:
(P2, No need to do this now) This function is starting to get a bit long. I'd consider breaking each section into helper functions that you call sequentially.

Reply:
Yep I have the same feeling. I'll add a note.